source/_templates/layout.html

```diff
@@ -171,6 +171,7 @@
 	})(window,document,'script','dataLayer','GTM-P9XQZ7X');</script>
 	<!-- End Google Tag Manager -->
 
+    <script src="{{ pathto('_static/bootstrap/bootstrap.min.js', 1) }}"></script>
```

Comment:
Can we add this only where needed cleanly?

At a minimum, mark it deferred/async.

Reply:
Marked async, since loading it from the plugin (the way it loads in the python plugin) will end up loading the ls each time the plugin is used